src/main/java/dk/cngroup/lentils/service/ImageService.java

```diff
@@ -89,13 +91,24 @@ public Resource loadAsResource(final String filename) {
         }
     }
 
-    public String getFilePathToSave(final MultipartFile file) {
+    private String getFileName(final MultipartFile file) {
         if (file.isEmpty()) {
             throw new FileNotFoundException();
         }
-        String fileName = file.getOriginalFilename();
-        FileUtils.createDirectory(getDirectory());
-        return getDirectory() + fileName;
+        return getFileString(file.getOriginalFilename());
+    }
+
+    public String getFileString(final String filename) {
```

Comment:
Z toho jména ani těla metody si nejsem jistý co to vlatně dělá.
Šlo by to nějak přejmenovat?

Reply:
Prejmenuju to na addHashToFilename - vezme to jmeno zadaneho souboru a prida k nemu aktualni datum + 4 pismenny hash.
Ted jsem jeste zjistila, ze nemam osetrene zadani souboru bez pripony (i kdyz je prednastavene zadavani obrazkovych souboru) - to jeste dodam